extensions/ql-vscode/src/vscode-tests/cli-integration/remote-queries/variant-analysis-manager.test.ts

```diff
@@ -780,7 +780,12 @@ describe("Variant Analysis Manager", async function () {
         });
 
         it("should remove variant analysis", async () => {
-          await variantAnalysisManager.onVariantAnalysisUpdated(
+          pathExistsStub
+            .withArgs(
+              path.join(storagePath, dummyVariantAnalysis.id.toString()),
+            )
+            .resolves(true);
+          await variantAnalysisManager.rehydrateVariantAnalysis(
```

Comment:
Why do we need to add this step?

Reply:
Previously, it would store the variant analysis in the `variantAnalyses` map when it was changed. Now, we only store it in the map if it's changed **and** it's already in the map. So, we need to make sure it's still in the map. This was done with the call to `onVariantAnalysisUpdated` before, but now you need to actually fully rehydrate the variant analysis.